pkg/admin/.openapi-generator/FILES

```diff
@@ -0,0 +1,31 @@
+.gitignore
```

Comment:
This is the file we were talking about earlier with Jordan that gets created by the odd openapigenerator. Need it tagged to `4.3.1`